tests/test_icml_conference.py

```diff
@@ -2961,7 +2967,136 @@ def test_rebuttal_stage(self, openreview_client, helpers):
             )
         )
 
-        helpers.await_queue(openreview_client)        
+        helpers.await_queue(openreview_client)
+
+        invitation = client.get_invitation(f'openreview.net/Support/-/Request{request_form.number}/Rebuttal_Stage')
+        invitation.cdate = openreview.tools.datetime_millis(datetime.datetime.utcnow())
+        client.post_invitation(invitation)
+
+        # post a rebuttal stage note
+        now = datetime.datetime.utcnow()
+        start_date = now - datetime.timedelta(days=2)
+        due_date = now + datetime.timedelta(days=3)
+        pc_client.post_note(openreview.Note(
+            content={
+                'rebuttal_start_date': start_date.strftime('%Y/%m/%d'),
+                'rebuttal_deadline': due_date.strftime('%Y/%m/%d'),
+                'number_of_rebuttals': 'Multiple author rebuttals per paper',
+                'email_program_chairs_about_rebuttals': 'No, do not email program chairs about received rebuttals'
```

Comment:
you are not selecting any readers here? are the readers optional?

Reply:
Yes, because PCs and authors are added always. This is in the description but I do think it might be a bit confusing. I thought about adding PCs and authors as default but we don't really need them to be there for the stage because they are always added by default.